ui/apps/platform/src/Containers/Clusters/DiscoveredClusters/DiscoveredClustersPage.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { ReactElement, useEffect, useState } from 'react';
+import {
+    Alert,
+    Breadcrumb,
+    BreadcrumbItem,
+    Bullseye,
+    Flex,
+    PageSection,
+    Spinner,
+    Text,
+    Title,
+} from '@patternfly/react-core';
+
+import BreadcrumbItemLink from 'Components/BreadcrumbItemLink';
+import PageTitle from 'Components/PageTitle';
+import useURLPagination from 'hooks/useURLPagination';
+import useURLSearch from 'hooks/useURLSearch';
+import useURLSort from 'hooks/useURLSort';
+import {
+    DiscoveredCluster,
+    countDiscoveredClusters,
+    defaultSortOption,
+    listDiscoveredClusters,
+    sortFields,
+} from 'services/DiscoveredClustersService';
+import { getAxiosErrorMessage } from 'utils/responseErrorUtils';
+import { clustersBasePath } from 'routePaths';
+
+import DiscoveredClustersTable from './DiscoveredClustersTable';
+import DiscoveredClustersToolbar from './DiscoveredClustersToolbar';
+
+const title = 'Discovered clusters';
+
+function DiscoveredClustersPage(): ReactElement {
+    const { page, perPage, setPage, setPerPage } = useURLPagination(10);
+    const { searchFilter, setSearchFilter } = useURLSearch();
+    const { getSortParams, sortOption } = useURLSort({ defaultSortOption, sortFields });
+
+    const [count, setCount] = useState(0);
+    const [errorMessage, setErrorMessage] = useState('');
+    const [clusters, setClusters] = useState<DiscoveredCluster[]>([]);
+    const [isLoading, setIsLoading] = useState(false);
+
+    useEffect(() => {
+        setIsLoading(true);
+
+        // const listArg = getListDiscoveredClustersArg({ page, perPage, searchFilter, sortOption });
+        // const { filter } = listArg;
+
+        Promise.all([countDiscoveredClusters(/* filter */), listDiscoveredClusters(/* listArg */)])
+            .then(([countArg, clustersArg]) => {
+                setClusters(clustersArg);
+                setCount(countArg);
```

Comment:
nit: I initial read the `...Arg` part of these var names as "Argument" which confused me. Maybe "Response" would be better suffix.

```suggestion
            .then(([countResponse, clustersResponse]) => {
                setClusters(clustersResponse);
                setCount(countResponse);
```

Reply:
Thank you for comments from reader viewpoint. I renamed with **FromResponse** suffix.